### ISSUE TITLE
Breaking chest under water triggers error in console

### ISSUE THREAD

**Issue by aylacullen** [PaperMC/Paper#4729]
### What behaviour is expected:
Find treasure chest

### What behaviour is observed:
Water

### Steps/models to reproduce:
Not sure how to reproduce

### Plugin list:
AnvilLot, Essentials, EssentialsChat, MineableSpawners, ProtocolLib, Vault, WorldEdit

### Paper version:
Paper version git-Paper-212 (MC: 1.16.3) (Implementing API version 1.16.3-R0.1-SNAPSHOT)

### Anything else:
Got a treasure map.  Went to location of X and started digging, first I dug 10 blocks down, then 30 block radius, including out into water.  Found nothing.  Used spectator mode to see if I maybe missed it; found nothing.  Got suspicious, checked server log, and saw error re: chest. Copied and pasted here:

[02:08:56 ERROR]: Block at -14,503, 32, -18,116 is Block{minecraft:water} but has net.minecraft.server.v1_16_R2.TileEntityChest@90cc00a. Bukkit will attempt to fix this, but there may be additional damage that we cannot recover.
[02:09:42 ERROR]: Block at -14,510, 33, -18,107 is Block{minecraft:water} but has net.minecraft.server.v1_16_R2.TileEntityChest@61a58252. Bukkit will attempt to fix this, but there may be additional damage that we cannot recover.

Let me know if this is the wrong place to report this.

**Comment by HexedHero** [comment]
This nice behaviour and happens to everyone, especially when using things such as WorldEdit and I see it happening sometimes when players just place checks in water and remove them later

**Comment by aylacullen** [comment]
I wasn't using WorldEdit.  WorldEdit plugin is installed, yes, but it's never used.  Aside from keeping the plugin updated, the plugin hasn't been used in over a year.  The server is a 100% survival server with no ops users; even the server admins exclusively play in survival without ops and only in survival mode (I'm one of those admins).  The only time ops is used is when a bug shows up, such as this one.

**Comment by HexedHero** [comment]
Yea, exactly the second part I said. I have the same server and many others do. I get this and it means nothing. Feel free to search the discord for the same things and people replying with it means nothing.

**Comment by aylacullen** [comment]
Well, in this case, it resulted in a treasure chest disappearing. 

I found a similar bug report by someone for Spigot: 
https://hub.spigotmc.org/jira/si/jira.issueviews:issue-html/SPIGOT-5560/SPIGOT-5560.html

Edit: I just realized the poster mentioned breaking chests underwater.  I remembered I broke two chests underwater and was not able to find the chest drops, so I just moved on.  So the error I saw had nothing to do with the treasure chest, and everything to do with missing drops as a result of breaking chests underwater.  I will now attempt to reproduce. 

OK, I am able to reproduce.  It generates an error in console every time a chest is broken underwater.  I have updated the issue title to reflect the current issue.
